trillian/ctfe/ct_server/main.go

```diff
@@ -40,13 +40,15 @@ import (
 
 // Global flags that affect all log instances.
 var (
-	httpEndpoint      = flag.String("http_endpoint", "localhost:6962", "Endpoint for HTTP (host:port)")
-	rpcBackendFlag    = flag.String("log_rpc_server", "localhost:8090", "Backend specification; comma-separated list or etcd service name (if --etcd_servers specified)")
-	rpcDeadlineFlag   = flag.Duration("rpc_deadline", time.Second*10, "Deadline for backend RPC requests")
-	logConfigFlag     = flag.String("log_config", "", "File holding log config in JSON")
-	maxGetEntriesFlag = flag.Int64("maxGetEntriesAllowed", 0, "Max number of entries we allow in a get-entries request (default 50)")
-	etcdServers       = flag.String("etcd_servers", "", "A comma-separated list of etcd servers")
-	etcdHTTPService   = flag.String("etcd_http_service", "trillian-ctfe-http", "Service name to announce our HTTP endpoint under")
+	httpEndpoint       = flag.String("http_endpoint", "localhost:6962", "Endpoint for HTTP (host:port)")
+	metricsEndpoint    = flag.String("metrics_endpoint", "", "Endpoint for serving metrics; if left blank, metrics will be visible on --http_endpoint")
```

Comment:
I guess we could have an empty `--metrics_endpoint` imply that no metrics should be exposed.

For now I'll just set the default value of the flag so it defaults to a different port.